crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -938,6 +942,31 @@ mod private {
             &self.events
         }
 
+        /// Find a single event in this room.
+        pub async fn event(
```

Comment:
Can we call this `find_event`? Also, can we add in the comment that it will try to load from memory first, and then attempt to load it from persisted storage, if it couldn't find it in memory?